### PR TITLE
Update component overview styles

### LINKED ISSUE #568: [Pages] Component Overview
### This is a requirement for Beta release.
This work include:
- [x] design & spec out Component overview card component
- [x] design & export simplified component images
Below is the current design with old components in them.

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #568

@jeanservaas  I think we need updated images with a background color for some of the components. Accordion looks like it disappears on hover. 😉 

https://deploy-preview-630--carbon-website.netlify.com/components/overview

### CODE REVIEW

**Comment by alisonjoseph** [comment]
All updated! just waiting on new images @jeanservaas 😄

**Comment by jeanservaas** [comment]
@alisonjoseph 

ok, whew, here are the images, had to take some of those svgs into illustrator to fix. svgs don't like masks.

https://ibm.box.com/s/yf7bsp5a9rn9yrjgjeh33h04598gpt9o

File: src/content/getting-started/about-carbon/about-carbon.md [hunk 1]
```diff
@@ -42,6 +42,4 @@ Carbon is the official implementation of the IBM Design Language for product and
 
 Carbon is built by developers, for developers. Our front-end libraries are continuously maintained and fully supported by the core Carbon team (and an army of open-source contributors).
 
-There are two different implementations of the Carbon Design System. One is a vanilla JavaScript library; that's written in ES6+, SCSS, and HTML and ships deliverables in vanilla JavaScript in ES5 as well as ES6, CSS, SCSS and HTML. The other is a React library that ships a React rewrite of the JavaScript, but uses the same CSS as the core library.
-
-There is also an unofficial, internal-only Angular library for Carbon Components that was built out by a team other than Carbon. Please note that the Carbon team does not maintain this repo.
+There are three different implementations of the Carbon Design System. One is a vanilla JavaScript library; that's written in ES6+, SCSS, and HTML and ships deliverables in vanilla JavaScript in ES5 as well as ES6, CSS, SCSS and HTML. Another is a React library that ships a React rewrite of the JavaScript, but uses the same CSS as the core library. There is also an Angular library that uses the same CSS.
```
Comment (jnm2377) [inline src/content/getting-started/about-carbon/about-carbon.md]: Random question, is the Vue version internal only or is that just not being publicized until it's more fleshed out?

Comment (alisonjoseph) [reply src/content/getting-started/about-carbon/about-carbon.md]: Yea, we are waiting to add it, hopefully soon though!

**Comment by alisonjoseph** [comment]
images are updated @jeanservaas! looks much better 😸

**Pull request merged**